.zenodo.json

```
@@ -1,7 +1,7 @@
 {
     "description": "Structure-Preserving Numerical Methods for Dispersive Shallow Water Models",
```

```suggestion
    "description": "A Julia library of structure-preserving numerical methods for dispersive wave equations",
```
Should we maybe make this consistent?